ui-kit/src/main/res/values/styles.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <style name="UiKit.Native.Snackbar" parent="Widget.MaterialComponents.Snackbar">
```

Comment:
hows the idea to name UiKit styles like this?

Reply:
Looks nice!